<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>士麦那播放器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ========== 主窗口：全局基础样式 ========== */
QWidget#MyWidget {
    margin: 0px;
    padding: 0px;
    border: none;
    background-color: #2b2b2b; /* 深灰底色，营造播放器质感 */
    min-width: 700px;
    min-height: 500px;
}

/* ========== 视频容器（widget）：纯黑背景 + 视觉分层 ========== */
/* 注意：此 widget 是你在 .ui 中用于承载 QVideoWidget 的普通 QWidget，命名为 &quot;widget&quot; */
QWidget#widget {
    margin: 0px;
    padding: 0px;
    border: none;
    background-color: black; /* 视频区纯黑，聚焦内容 */
    /* 关键：不设置 height / max-height / min-height，交由布局管理器控制 */
    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3); /* 轻微阴影，区分控件区 */
}

/* ========== 下方控件区容器（verticalLayoutWidget） ========== */
/* 对应 Qt Designer 中 verticalLayout 的父 widget */
QWidget#verticalLayoutWidget {
    margin: 0px;
    padding: 5px 8px; /* 轻微内边距，避免控件贴边 */
    border: none;
    background-color: #333333; /* 稍浅灰，视觉分层 */
    /* 不固定高度！由主布局 stretch factor 决定 */
    z-index: 999;
}

/* ========== 按钮：统一风格+弹性适配 ========== */
QPushButton {
    min-height: 32px;
    max-height: 36px;
    min-width: 70px;
    padding: 4px 8px;
    border: 1px solid #555555;
    border-radius: 5px;
    background-color: #444444;
    color: #f0f0f0;
    font-size: 12px;
    text-align: center;
    transition: all 0.2s ease;
}

QPushButton:hover:enabled {
    border: 1px solid #4da6ff;
    background-color: #555555;
    color: #4da6ff;
}

QPushButton:pressed:enabled {
    border: 1px solid #3a8bd8;
    background-color: #383838;
    padding: 5px 7px 3px 9px;
}

QPushButton:disabled {
    min-height: 32px;
    max-height: 36px;
    min-width: 70px;
    border: 1px solid #666666;
    background-color: rgba(68, 68, 68, 0.5);
    color: #999999;
    border-radius: 5px;
    padding: 4px 8px;
}

/* ========== 滑块：简洁精致+适配深色风格 ========== */
QSlider {
    min-height: 28px;
    max-height: 32px;
    margin: 0px 5px;
    border: none;
}

QSlider::groove:horizontal {
    height: 4px;
    background-color: rgba(100, 100, 100, 0.5);
    border-radius: 2px;
    margin: 0px 3px;
}

QSlider::handle:horizontal {
    width: 14px;
    height: 14px;
    background-color: #4da6ff;
    border-radius: 7px;
    margin: -5px 0px;
    transition: transform 0.2s ease;
}

QSlider::handle:horizontal:hover {
    transform: scale(1.1);
}

/* ========== 标签：弱化不抢镜+清晰可读 ========== */
QLabel {
    margin: 0px 5px;
    padding: 3px 6px;
    border: none;
    font-size: 12px;
    color: #f0f0f0;
    background-color: rgba(0, 0, 0, 0.2);
    border-radius: 4px;
    min-height: 22px;
    max-height: 24px;
}

/* ========== 播放列表表格（可选美化） ========== */
QTableView {
    background-color: #2d2d2d;
    alternate-background-color: #353535;
    color: #e0e0e0;
    gridline-color: #444444;
    selection-background-color: #4da6ff;
    selection-color: white;
    border: 1px solid #555555;
}

QHeaderView::section {
    background-color: #3a3a3a;
    color: #cccccc;
    padding: 4px;
    border: 1px solid #444444;
    font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>374</width>
     <height>586</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btStart">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btReset">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btLast">
        <property name="text">
         <string>上一个视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btNext">
        <property name="text">
         <string>下一个视频</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="light">
          <property name="text">
           <string>亮度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lights">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="minimum">
           <number>-10</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btUpload">
        <property name="text">
         <string>上传视频</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="time">
        <property name="text">
         <string>时长：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="times">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TickPosition::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totaltime">
        <property name="text">
         <string>总时长</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btList">
      <property name="text">
       <string>播放列表</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btExport">
        <property name="text">
         <string>导出播放列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btClose">
        <property name="text">
         <string>退出播放器</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
